src/main/java/br/ufpe/cin/logging/LoggerStatistics.java

```diff
@@ -161,21 +161,21 @@ private static void logSummary() throws IOException{
 				for(int i = 1; i <lines.size(); i++){
 					String[] columns = lines.get(i).split(",");
 
-					ssmergeconfs += Integer.valueOf(columns[2]);
-					ssmergeloc 	 += Integer.valueOf(columns[3]);
-					ssmergerenamingconfs += Integer.valueOf(columns[4]);
-					ssmergedeletionconfs += Integer.valueOf(columns[5]);
-					ssmergetaeconfs   += Integer.valueOf(columns[7]);
-					ssmergenereoconfs += Integer.valueOf(columns[8]);
-					ssmergeinitlblocksconfs += Integer.valueOf(columns[9]);
-					ssmergeacidentalconfs 	+= Integer.valueOf(columns[10]);
-					unmergeconfs += Integer.valueOf(columns[11]);
-					unmergeloc 	 += Integer.valueOf(columns[12]);
+					ssmergeconfs += Integer.parseInt(columns[2]);
```

Comment:
Qual o motivo de trocar `valueOf` por `parseInt`?

Reply:
Esse foi um bug de performance detectado.

Reply:
Pode explicar melhor o bug? Fique curioso

Reply:
acho que não é bem bug. é uma recomendação da ferramenta de análise de código.

Reply:
certo @jvcoutinho?

Reply:
Esse foi o output da ferramenta:
`[Performance | DM_BOXED_PRIMITIVE_FOR_PARSING] br.ufpe.cin.logging.(LoggerStatistics.java:164)  [priority 1]
  A boxed primitive is created from a String, just to extract the unboxed primitive value.
  It is more efficient to just call the static parseXXX method.`
É um bug de prioridade 1, então ele quebra a build.

Reply:
@guilhermejccavalcanti , mas esse não é um caso em que apenas precisamos do valor primitivo?

Reply:
isso @gaabs , foi o que disse aqui "parseInt é recomendado quando a gente só precisa do valor primitivo, sem necessidade dos métodos extras encapsulados pela classe Integer."